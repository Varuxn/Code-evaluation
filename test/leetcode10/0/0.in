10 10
0 7
4 6
1 2
6 9
2 4
0 1
4 8
2 3
2 5
7 3
