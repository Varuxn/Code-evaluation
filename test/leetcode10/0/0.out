2 4 2 5 4 6 4 8 6 9 